<ui version="4.0" >
 <class>processDialog</class>
 <widget class="QDialog" name="processDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>670</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>1020</width>
    <height>670</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Data processing</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="minimumSize" >
      <size>
       <width>70</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>70</width>
       <height>25</height>
      </size>
     </property>
     <property name="text" >
      <string>Data Table</string>
     </property>
    </widget>
   </item>
   <item rowspan="2" row="0" column="1" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="minimumSize" >
      <size>
       <width>550</width>
       <height>110</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>650</width>
       <height>110</height>
      </size>
     </property>
     <property name="title" >
      <string>Control panel</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" >
       <layout class="QVBoxLayout" name="verticalLayout_9" >
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8" >
          <item>
           <widget class="QLabel" name="indicator_label" >
            <property name="minimumSize" >
             <size>
              <width>100</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>100</width>
              <height>20</height>
             </size>
            </property>
            <property name="text" >
             <string>NOT RUNNING</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" >
          <item>
           <widget class="QLabel" name="channels_label" >
            <property name="minimumSize" >
             <size>
              <width>60</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>60</width>
              <height>20</height>
             </size>
            </property>
            <property name="text" >
             <string>Channels:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" >
            <property name="sizeConstraint" >
             <enum>QLayout::SetFixedSize</enum>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2" >
              <property name="sizeConstraint" >
               <enum>QLayout::SetFixedSize</enum>
              </property>
              <item>
               <widget class="QLabel" name="channel1_label" >
                <property name="text" >
                 <string>Ch1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="channel1_box" >
                <property name="text" >
                 <string/>
                </property>
                <property name="checked" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3" >
              <property name="sizeConstraint" >
               <enum>QLayout::SetFixedSize</enum>
              </property>
              <item>
               <widget class="QLabel" name="channel2_label" >
                <property name="text" >
                 <string>Ch2</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="channel2_box" >
                <property name="text" >
                 <string/>
                </property>
                <property name="checked" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4" >
              <property name="sizeConstraint" >
               <enum>QLayout::SetFixedSize</enum>
              </property>
              <item>
               <widget class="QLabel" name="channel3_label" >
                <property name="text" >
                 <string>Ch3</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="channel3_box" >
                <property name="text" >
                 <string/>
                </property>
                <property name="checked" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5" >
              <property name="sizeConstraint" >
               <enum>QLayout::SetFixedSize</enum>
              </property>
              <item>
               <widget class="QLabel" name="channel4_label" >
                <property name="text" >
                 <string>Ch4</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="channel4_box" >
                <property name="text" >
                 <string/>
                </property>
                <property name="checked" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6" >
              <property name="sizeConstraint" >
               <enum>QLayout::SetFixedSize</enum>
              </property>
              <item>
               <widget class="QLabel" name="channel5_label" >
                <property name="text" >
                 <string>Ch5</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="channel5_box" >
                <property name="text" >
                 <string/>
                </property>
                <property name="checked" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7" >
              <property name="sizeConstraint" >
               <enum>QLayout::SetFixedSize</enum>
              </property>
              <item>
               <widget class="QLabel" name="channel6_label" >
                <property name="text" >
                 <string>Ch6</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="channel6_box" >
                <property name="text" >
                 <string/>
                </property>
                <property name="checked" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="1" >
       <layout class="QHBoxLayout" name="horizontalLayout_4" >
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" >
          <item>
           <widget class="QLabel" name="zone_length_label" >
            <property name="maximumSize" >
             <size>
              <width>115</width>
              <height>19</height>
             </size>
            </property>
            <property name="text" >
             <string>X points per screen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="zone_length_spin" >
            <property name="minimumSize" >
             <size>
              <width>120</width>
              <height>21</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>120</width>
              <height>21</height>
             </size>
            </property>
            <property name="minimum" >
             <number>10</number>
            </property>
            <property name="maximum" >
             <number>1000000</number>
            </property>
            <property name="singleStep" >
             <number>10</number>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" >
            <item>
             <widget class="QLabel" name="scale_label" >
              <property name="minimumSize" >
               <size>
                <width>85</width>
                <height>19</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>85</width>
                <height>19</height>
               </size>
              </property>
              <property name="text" >
               <string>Current Scale:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="scale_value_label" >
              <property name="minimumSize" >
               <size>
                <width>45</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>45</width>
                <height>20</height>
               </size>
              </property>
              <property name="text" >
               <string>1</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item rowspan="2" row="1" column="0" >
    <widget class="QTableWidget" name="display_table" >
     <property name="minimumSize" >
      <size>
       <width>320</width>
       <height>520</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>320</width>
       <height>540</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Tab 1</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout" >
       <item row="0" column="0" >
        <widget class="QwtPlot" name="plot" >
         <property name="minimumSize" >
          <size>
           <width>300</width>
           <height>300</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>650</width>
           <height>650</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QPushButton" name="stopButton" >
     <property name="text" >
      <string>S&amp;tart</string>
     </property>
     <property name="autoDefault" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
